### DISCUSSION TITLE
Hosting saleor in a remote machine

### DISCUSSION THREAD

**Discussion by manuelaguirre** [saleor/saleor#8505] [Deployment]
I need to host saleor in a server, exposing port 8000, what is the best way to do it? If I install saleor-platform I get a CORS message when I press the login page, because the dashboard thinks the api is on `localhost` while it actually should be at `192.1.1.xxx` for instance. I have found no way to circunvent this, nor to install the services separately

**Comment by karolkielecki** [comment]
@manuelaguirre Hi! The `saleor-platform` repo hasn't been designed for deployment as stated in `readme`. Saleor's referred deployment method is Docker. Storefront and dashboard are static apps and they don't require additional servers. You can use AWS S3 for them. 

You can find more information here: https://docs.saleor.io/docs/3.0/developer/running-saleor/docker

**Reply by manuelaguirre** [reply]
Thank you Karol! What should I do after building the image? I've tried running a container with it but after telling me that the application has started I'm not able to ping it in port 8000 (Neither from localhost nor network)

**Reply by manuelaguirre** [reply]
Oh so I forgot to link ports when running container. Sorry about that. Where should I declare `ALLOWED_HOSTS` I've tried to export it (I'm on Ubuntu) but should I build again?

**Reply by marcorocci** [reply]
same problem here.

**Reply by karolkielecki** [reply]
There is env variable `ALLOWED_HOSTS` https://docs.saleor.io/docs/2.11/developer/running-saleor/configuration#allowed_hosts

**Reply by marcorocci** [reply]
it seems not to work:
https://github.com/saleor/saleor/discussions/8537

**Reply by manuelaguirre** [reply]
So if my server is hosted on 192.168.1.53:8000 and my dashboard is served from 192.168.1.53:9000, what should I put on env variables?

**Reply by lastmaj** [reply]
@manuelaguirre try 192.168.1.53

**Reply by l4b4r4b4b4** [reply]
any luck? :)
